err.c

```diff
@@ -26,6 +26,36 @@ void (*err_exit)(int) = exit;
 const char *err_line_ending = "\n";
 int err_flags = 0;
 
+/* Parse the BST_VERBOSITY environment variable to determine the requested
+   verbosity. Higher numbers are associated with more verbosity, with 9
+   being the most verbose and 0 being the least verbose. The default
+   verbosity level is 1. */
+
+static int parsedverbosity()
```

Comment:
The definition needs to be `static int parsedverbosity(void)` (note the void), otherwise you can pass any number of arguments; it's the syntax when you're pulling things out with the calling convention (and it's mostly used by the syscall(3) function these days)